tests/Dockerfile

```diff
@@ -0,0 +1,43 @@
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2018 Red Hat, Inc.
+#
+
+FROM fedora:27
+
+MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
+
+ARG DOCKER_PREFIX=kubevirt
+ARG DOCKER_TAG=latest
+ARG NAMESPACE=kube-system
+
+ENV WORKSPACE=/kubevirt-testing
+ENV DATA_DIR=${WORKSPACE}/data
+ENV RESULTS_DIR=${DATA_DIR}/results
+ENV KUBECONFIG=${DATA_DIR}/kubeconfig
+
+WORKDIR "${WORKSPACE}"
+
+VOLUME ["${DATA_DIR}"]
+
+RUN dnf install -y origin-clients && dnf clean all
```

Comment:
Hm, might get tricky to catch up with having compatible clients, depending on which version kubevirt is  deployed on, or what is inside the manifests ... I would prefere if we can do everything directly with our binary, which in includes client code which by definition should be compatible, since if our client-go is not compatible in the tests, we simply don't support the cluster under test.